src/styles/hero.css

```diff
@@ -5,9 +5,12 @@
 }
 
 .hero h1 {
-  font-size: 80px;
-  text-shadow: 0 7px rgba(51, 0, 101, 1);
-  margin: 0 0 60px 0;
+  font-family: Roboto-Bold;
```

Comment:
I think this needs to be `Roboto` and not `Roboto-Bold` (or else there's some other issue).

Changing to just `Roboto` fixes it though:

<img width="1047" alt="Screen Shot 2021-10-12 at 10 58 42 AM" src="https://user-images.githubusercontent.com/110551/137006286-445e8622-0263-4bf7-821b-44aaddddb27f.png">